openstack/identity/v3/applicationcredentials/results.go

```diff
@@ -0,0 +1,127 @@
+package applicationcredentials
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+type Role struct {
+	// DomainID is the domain ID the role belongs to.
+	DomainID string `json:"domain_id,omitempty"`
```

Comment:
It doesn't look like `domain_id` is specified in the schema: https://github.com/openstack/keystone/blob/ef87a202127f0ac84b857716e72595ce75058378/keystone/application_credential/schema.py#L22-L25

But I'm seeing it in the API output, so it's a legitimate field.

Nothing to do here, just a note for posterity.